.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
 
 before_install:
   - phpenv config-add project-base/docker/php-fpm/php-ini-overrides.ini
+  - echo '' | pecl install redis-4.1.1
```

Comment:
what about .travis.yml for package framework and all the packages that uses framework ?

Reply:
what does this do? i think it seems that it just use the default value for some question in interactive mode yes ?
i would maybe put the thing that we skip or install additionaly in comment

Reply:
I don't think it related to PHP 7.1 != PHP 7.1.0.
This is the only Travis script where the Redis extension is relevant, I think. It's the only one where the full DIC of the application is built (and lazy loading is working only in the newer version).
The `echo '' |` is for non-interactive PECL installation.

Reply:
What do you suggest to do?

Reply:
hmmm, there are skipped 2 thing 
```
enable igbinary serializer support? [no] 
enable lzf compression support? [no]
```
i have already [PR](#1007) for improving travis check for monorepo so i am OK with this right now then.